setup.py

```diff
@@ -47,7 +47,7 @@
         "six>=1.9.0,<2.0.0",
         "sortedcontainers>=1.5.9<3.0.0",
         "statsd>=3.0.0,<4.0.0",
-        "urllib3>=1.22,<1.26",
+        "urllib3>=1.22,<2.0.0",
```

Comment:
Updated to match https://github.com/lyft/flytekit/pull/155 to avoid merge conflict.